Makefile

```
@@ -1,13 +1,19 @@
-.PHONY: help fmt lint test test-verbose check build
+VERSION := $(shell cat VERSION)
+
+.PHONY: help fmt lint test test-verbose check build docker-build db push increment-version get-version
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)